pkg/klusterlet/clusterinfo/controller.go

```diff
@@ -84,7 +85,7 @@ func (r *ClusterInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			claimLister:          r.ClaimLister,
 		},
 	}
-	if !r.DisableLoggingInfoSyncer {
+	if !r.DisableLoggingInfoSyncer || r.InstallMode != "Hosted" {
```

Comment:
curious: when do we set DisableLoggingInfoSyncer to true?

Reply:
the controller is imported in  standalone controlplane, and started with DisableLoggingInfoSyncer == true.

Reply:
DisableLoggingInfoSyncer is not a flag of agent binary, let me use it to replace InstallMode.